examples/llm_plus_gnn/nvtx_webqsp_example.py

```diff
@@ -0,0 +1,22 @@
+import argparse
+
+import torch
+
+from torch_geometric.datasets import web_qsp_dataset
+from torch_geometric.profile import nvtxit
+
+# Apply Patches
+web_qsp_dataset.retrieval_via_pcst = nvtxit()(
+    web_qsp_dataset.retrieval_via_pcst)
+web_qsp_dataset.WebQSPDataset.process = nvtxit()(
```

Comment:
why do we have two of these, one for my old webqsp and one for ur new one. in general for this PR to be merged there should be only one webqsp, and associated example